tools/Modules/RFC/RFC.psm1

```diff
@@ -0,0 +1,369 @@
+$script:repoBase = "https://api.github.com/repos"
```

Comment:
Please add the Microsoft copyright header

Reply:
done